include/industry_standard/spdm.h

```diff
@@ -16,6 +16,8 @@
 
 /* 3 means SPDM spec 1.0, 1.1, 1.2 */
 #define SPDM_MAX_VERSION_COUNT 3
+/* 2 means SPDM secured message version 1.0, 1.1 */
+#define SPDM_MAX_SECURED_VERSION_COUNT 2
```

Comment:
This should go in https://github.com/DMTF/libspdm/blob/main/include/industry_standard/spdm_secured_message.h

Reply:
And please name it `SECURED_SPDM_MAX_VERSION_COUNT`.

Reply:
Thanks. I have changed the code.